sortinghat/core/db.py

```diff
@@ -196,6 +197,44 @@ def find_domain(domain_name):
         return domain
 
 
+def find_recommenderExclusionList(exclusion):
```

Comment:
I don't think we need this function because add/remove functions should try to do their job even if the term doesn't exist.